system-tests/tests/svc-bootcfg

```diff
@@ -5,37 +5,46 @@ source ${SYSTESTS_PRELUDE} && prepare
 
 set -x -e -o pipefail
 
-# Arrange
-cobbler distro add --name fake --arch x86_64 --kernel ${fake_kernel} \
-	--initrd ${fake_initramfs}
-cobbler profile add --name fake --distro fake
-cobbler system add --name testbed --profile fake
+distro=esxi70
```

Comment:
Can we maybe add a second test for this case? I would desire to test both cases. The "standard one" I had before and the ESXi case you added here. I would propose just copy the file you have now and suffix it with `-esxi`, then you revert the changes in this file here.

Reply:
I will do that, but I will not be able keep the exact previous test without some minor modifications, as `bootcfg.template` has changed.